static/src/stylesheets/module/site-messages/_engagement-banner.scss

```diff
@@ -123,12 +123,16 @@
     display: inline-block;
     vertical-align: middle;
 
-    height: auto;
-    width: 114px;
+    height: 17px;
 
     @include mq(tablet) {
         display: block;
-        width: 121px;
+        height: 18px;
```

Comment:
it was using the width to scale it before, but now that it's a row rather than a single item it makes more sense to scale using the height.  otherwise it ends up on two rows.